e2e-test/src/test/java/io/cdap/plugin/normalize/stepsdesign/NormalizeTransform.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.normalize.stepsdesign;
+
+import io.cdap.e2e.pages.actions.CdfPipelineRunAction;
+import io.cdap.e2e.pages.actions.CdfPluginPropertiesActions;
+
+import io.cdap.e2e.utils.AssertionHelper;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.WaitHelper;
+import io.cdap.plugin.normalize.actions.NormalizeActions;
+import io.cdap.plugin.normalize.locators.NormalizeLocators;
+import io.cdap.plugin.utils.E2ETestConstants;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import stepsdesign.BeforeActions;
+
+import java.io.IOException;
+
+/**
+ * Normalize Plugin related step design.
+ */
+public class NormalizeTransform implements CdfHelper {
+  private static final Logger logger = LoggerFactory.getLogger(CdfPluginPropertiesActions.class);
+
+  @Then("Delete the output schema added")
+  public void deleteTheOutputSchemaAdded() {
+    NormalizeActions.clickDeleteField(0);
+  }
+
+  @Then("Verify Normalize plugin in-line error message for incorrect field mapping: {string}")
```

Comment:
generic common method ->

Verify Normalize plugin property: {string}  in-line error message for value: {string}

Usage in scenario ->  Verify Normalize plugin property: "fieldMapping" in-line error message for value: "mappingField"

Use if else based on plugin property in method

Reply:
Need to discuss on this.